tests/codegen/wasm_exceptions.rs

```diff
@@ -1,5 +1,5 @@
 //@ only-wasm32
-//@ compile-flags: -C panic=unwind
+//@ compile-flags: -C panic=unwind -Z emscripten-wasm-eh
```

Comment:
Question: how does `-Z emscripten-wasm-eh` affect non-`emscripten` wasm32 targets?

Reply:
It's ignored on anywhere but emscripten.

Reply:
Alright (I find that behavior... interesting, but given it's an unstable flag, whatever).